src/block_cache/block_io.c

```diff
@@ -150,6 +150,16 @@ __wt_blkcache_read(WT_SESSION_IMPL *session, WT_ITEM *buf, WT_PAGE_BLOCK_META *b
             *block_meta = block_meta_tmp;
 
         dsk = ip->data;
+
+        /*
```

Comment:
We should also add the stats to `__wt_blkcache_read_multi`

Reply:
I'm thinking that we can treat a page and its (possibly multiple) deltas as one thing and just increment the statistics once. Does this sound reasonable or do we care about how many deltas we are reading?